Fri Feb 10 2006 Stuart Hughes <stuarth@freescale.com>

This patch fixes the many warnings:

warning: #warning "You should include <sys/socket.h>. This time I will do it for you."


diff --exclude CVS -uNr iproute2-2.6.11-050330/include/linux/netlink.h iproute2-2.6.11-050330.modified/include/linux/netlink.h
--- iproute2-2.6.11-050330/include/linux/netlink.h	2005-04-01 20:58:11.000000000 +0100
+++ iproute2-2.6.11-050330.modified/include/linux/netlink.h	2006-02-10 16:54:14.000000000 +0000
@@ -1,7 +1,7 @@
 #ifndef __LINUX_NETLINK_H
 #define __LINUX_NETLINK_H
 
-#include <linux/socket.h> /* for sa_family_t */
+#include <sys/socket.h> /* for sa_family_t */
 #include <linux/types.h>
 
 #define NETLINK_ROUTE		0	/* Routing/device hook				*/
